### ISSUE TITLE
Any command regarding SES sends me the same response.

### ISSUE THREAD

**Issue by Pehesi97** [localstack/localstack#909]
`aws --endpoint-url=http://localhost:4579 ses create-configuration-set --configuration-set Name=Example`

```
Unable to parse response (syntax error: line 1, column 54), invalid XML received:
b'<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">\n<title>500 Internal Server Error</title>\n<h1>Internal Server Error</h1>\n<p>The server encountered an internal error and was unable to complete your request.  Either the server is overloaded or there is an error in the application.</p>\n'
```

`aws --endpoint-url=http://localhost:4579 ses get-send-statistics`

```
Unable to parse response (syntax error: line 1, column 54), invalid XML received:
b'<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">\n<title>500 Internal Server Error</title>\n<h1>Internal Server Error</h1>\n<p>The server encountered an internal error and was unable to complete your request.  Either the server is overloaded or there is an error in the application.</p>\n'
```

My configs and credentials are OK.

**Comment by adriangalera** [comment]
Same here

**Comment by orende** [comment]
Bumping this to ask the same thing. Getting the information from the get-send-statistics API call would be very useful in tests.

**Comment by whummer** [comment]
Thanks for reporting @Pehesi97 @adriangalera @orende . This should be fixed in #2946 - can you please give it a try with the `latest` version? Please report here if the problem persists. Thanks!

**Comment by mnfusaro** [comment]
Hi, i'm using the boto3 1.14.6 version and still get `NotImplementedError: The get_send_statistics action has not been implemented`
